resolver/resolver.cc

```diff
@@ -2062,6 +2062,12 @@ class ResolveSanityCheckWalk {
         ENFORCE(false, "These should have all been removed: {}", original->toString(ctx));
         return original;
     }
+    unique_ptr<ast::Expression> postTransformUnresolvedIdent(core::MutableContext ctx,
+                                                             unique_ptr<ast::UnresolvedIdent> original) {
+        ENFORCE(original->kind != ast::UnresolvedIdent::Local, "{} should have been removed by local_vars",
+                original->toString(ctx));
+        return original;
+    }
```

Comment:
This is just part of the sanity check, so I can move the `ENFORCE` that used to be in the namer here.